packages/create-svelte-ux/bin.js

```diff
@@ -107,6 +107,15 @@ const template = options_cli.template
 if (p.isCancel(template)) {
   process.exit(1);
 }
+const svelteNext = await p.confirm({
+  message: 'Would you like to use Svelte 5 (unstable)?',
+  initialValue: false,
+});
+if (p.isCancel(svelteNext)) {
+  process.exit(1);
+}
+const svelteVersion = svelteNext ? '^5.0.0-next.148' : '^4.2.17';
```

Comment:
```suggestion
const svelteVersion = svelteNext ? 'next' : 'latest';
```

Reply:
![image](https://github.com/techniq/svelte-ux/assets/177476/ab6b11d0-268c-40a1-aa59-05fd8e79aa15)

I like this mostly to make sure everyone is getting the latest releases of 4.x and 5.x without having to do a release every day.  The user can always hard code it later.

Thanks!